model/client4.go

```diff
@@ -4079,6 +4079,50 @@ func (c *Client4) MigrateAuthToSaml(fromAuthService string, usersMap map[string]
 	return CheckStatusOK(r), BuildResponse(r)
 }
 
+// UploadLdapPublicCertificate will upload a public certificate for LDAP and set the config to use it.
+// The filename parameter is deprecated and ignored: the server will pick a hard-coded filename when writing to disk.
+func (c *Client4) UploadLdapPublicCertificate(data []byte, filename string) (bool, *Response) {
```

Comment:
Do we need to include `filename` as an arg here? Since this is not breaking the API at all.